examples/http-server/main_test.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"net/http/httptest"
+	"strconv"
+	"testing"
+)
+
+func corazaGetRequest(getPath string) (int, string) {
+	if err := setupCoraza(); err != nil {
+		panic(err)
+	}
+	testServer := httptest.NewServer(corazaRequestHandler(http.HandlerFunc(hello)))
+	defer testServer.Close()
+
+	// Performs a get request
+	resp, err := http.Get(testServer.URL + getPath)
+	if err != nil {
+		log.Fatalln(err)
+	}
+	if resp.StatusCode == http.StatusOK {
+		bodyBytes, err := io.ReadAll(resp.Body)
+		if err != nil {
+			log.Fatal(err)
+		}
+		bodyString := string(bodyBytes)
+		return resp.StatusCode, bodyString
```

Comment:
Do we need to return the body? I don't see it being used.

Reply:
Nope, abstracting the function I added it "just in case", but, well...removed 😅